feature/client/src/main/java/com/mifos/feature/client/clientList/presentation/ClientListScreen.kt

```diff
@@ -98,11 +94,31 @@ internal fun ClientListScreen(
         viewModel.getClientList()
```

Comment:
duplicate. remove it, we already have that in viewmodel

Reply:
yeah thats right, its a duplicate